### PR TITLE
Fix Footer background color

### PR DESCRIPTION
**Pull request by tmansuroglu**
Footer background color changes based on route.

### CODE REVIEW

**Review by YamanSammani — APPROVED** [review]

**Comment by louisrli** [comment]
tests

**Comment by YamanSammani** [comment]
> tests

@louisrli , PTAL. Thanks.

**Review by louisrli — APPROVED** [review]

File: src/components/Footer/index.jsx [hunk 1]
```diff
@@ -7,15 +7,29 @@ import {
...
-import { Link } from "react-router-dom";
+import { Link, useLocation } from "react-router-dom";
 import { useTranslation } from "react-i18next";
 import globe from "./../../images/earthNav.svg";
 
 const Footer = () => {
+  const curRoute = useLocation().pathname;
   const [t] = useTranslation();
+  const shouldUseLightGreenBackground =
+    curRoute === "/about" ||
```
Comment (louisrli) [inline src/components/Footer/index.jsx]: LIGHT_GREEN_BACKGROUND_PAGES = ["/about", ...]

File: src/components/Footer/index.jsx [hunk 2]
```diff
@@ -7,15 +7,29 @@ import {
...
+    curRoute === "/games/memorygame" ||
+    curRoute === "/games/puzzlegame" ||
+    curRoute === "/games/quizgame";
 
   return (
-    <div className="footerContainer">
+    <div
+      className="footerContainer"
+      style={
+        shouldUseLightGreenBackground
```
Comment (louisrli) [inline src/components/Footer/index.jsx]: LIGHT_GREEN_BACKGROUND_PAGES.includes(curRoute)

**Review by louisrli — CHANGES_REQUESTED** [review]
fix magic string

File: src/components/Footer/index.jsx (hunk 1 above)
Comment (tmansuroglu) [reply src/components/Footer/index.jsx]: PTAL

File: src/components/Footer/index.jsx (hunk 2 above)
Comment (tmansuroglu) [reply src/components/Footer/index.jsx]: PTAL

**Comment by tmansuroglu** [comment]
I wonder when I will stop making this mistake.

**Review by louisrli — APPROVED** [review]

File: src/components/Footer/index.jsx [hunk 3]
```diff
@@ -7,15 +7,35 @@ import {
...
+  const curRoute = useLocation().pathname;
   const [t] = useTranslation();
 
   return (
-    <div className="footerContainer">
+    <div
+      className="footerContainer"
+      style={
+        LIGHT_GREEN_BACKGROUND_PAGES.includes(curRoute)
+          ? { backgroundColor: LIGHT_GREEN }
```
Comment (louisrli) [inline src/components/Footer/index.jsx]: {{ backgroundColor: x ? y : z }}

File: src/components/Footer/index.jsx [hunk 4]
```diff
@@ -7,15 +7,35 @@ import {
...
+const LIGHT_GREEN_BACKGROUND_PAGES = [
+  "/about",
+  "/resources",
+  "/games/memorygame",
+  "/games/puzzlegame",
+  "/games/quizgame",
+];
+
+const LIGHT_GREEN = "#b4d0c4";
+
```
Comment (louisrli) [inline src/components/Footer/index.jsx]: delete this newline. these constants are logically grouped together

File: src/components/Footer/index.jsx (hunk 3 above)
Comment (tmansuroglu) [reply src/components/Footer/index.jsx]: PTAL

File: src/components/Footer/index.jsx (hunk 4 above)
Comment (tmansuroglu) [reply src/components/Footer/index.jsx]: PTAL

**Comment by tmansuroglu** [comment]
@Ammar-64 can you merge?

**Comment by tmansuroglu** [comment]
@Ammar-64 ready for merge

**Comment by tmansuroglu** [comment]
@Ammar-64 ready for merge

**Comment by tmansuroglu** [comment]
@Ammar-64 why do you hate this PR ? :D

**Review by majdajroudi — APPROVED** [review]
lgtm

**Pull request merged**